thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dto/AnomalyFunctionDTO.java

```diff
@@ -61,7 +59,7 @@ public void setTopicMetric(String topicMetric) {
    * true if this function should get total metric
    */
   public boolean isToCalculateGlobalMetric() {
-    return StringUtils.isNotEmpty(this.getGlobalMetric());
+    return org.apache.commons.lang3.StringUtils.isNotEmpty(this.getGlobalMetric());
```

Comment:
strike off! --org.apache.commons.lang3.--

Reply:
There are two StringUtils used in the class: one is from ThirdEye and one is from apache. I'm not able to strike out org.apache.commons.lang3